src/components/Slider/slider.module.scss

```diff
@@ -1,18 +1,18 @@
 @function strip-units($number) {
-    @return $number / ($number * 0 + 1);
+    @return calc(#{$number} / calc(#{$number} * 0 + 1));
```

Comment:
is this performing a sass calc()? presuming yes.

Reply:
i looked it up. this performs a css calc :)

Reply:
ya, sass uses the same calc() it could also be math.div - but I prefer calc.

Reply:
I'm going to revert this one and see if the linter lets it slide :) looks like this one is busted.

Reply:
works without the `#{}` updating now.